images/pull-kubernetes-bazel/runner

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+git clone https://github.com/kubernetes/test-infra
+./test-infra/jenkins/bootstrap.py \
+    --upload='gs://kubernetes-jenkins/pr-logs' \
+    --repo=k8s.io/kubernetes \
+    --pull=${PULL_REFS} \
```

Comment:
Right, that makes sense, but I'm not sure if we want to keep this pattern or move away from it... The benefit is that it is portable but the downside is hard to understand.